.github/workflows/update_debian_repo_script.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+
+# to run this script as part of other update scripts, source the directory like below
+# source $GITHUB_WORKSPACE/.github/workflows/update_debian_repo_script.sh
+
+# you must have already set corresponding values for armhf_webPackages, arm64_webPackages, or all_webPackages
+# The corresponding appname in pi-apps will have its corresponding filepath= variable update
+# the filepath variable will contain the full filepath of the debian package with the version included
+
+# make sure required variable(s) is(are) supplied by the sourcing script
+if [ -n "$armhf_webPackages" ] || [ -n "$arm64_webPackages" ] || [ -n "$all_webPackages" ]; then
+#iterate through all variables supplied if an array
+# get longest arrary of passed variables
+num=$(echo -e "${#armhf_webPackages[@]}\n${#arm64_webPackages[@]}\n${#all_webPackages[@]}" | sort -n -r | head -1)
+
+for (( iter=0; iter<$num; iter++ )); do
+#make array appear as one variable inside for loop
+armhf_webPackages=${armhf_webPackages[$iter]}
+arm64_webPackages=${arm64_webPackages[$iter]}
+all_webPackages=${all_webPackages[$iter]}
```

Comment:
I was not aware this was possible: to reuse the same name for both an array and a variable, without encountering complexities. Where did you learn about this, @theofficialgman?

Reply:
I'm taking advantage of how bash uses variables and arrays.
the array doesn't get overwritten, only the first value in the array gets overwritten.

when you call the array without specifying the index, its assumes the first always.

so what is happening is I overwrite the first index of the array every time I move through the loop with the desired content of a later index